### PR TITLE
Server api 01

### PR DESCRIPTION
**Pull request by Ta1265**

### CODE REVIEW

**Review by JPK312 — APPROVED** [review]
DB schemas and APis look good

File: database/seed.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  const promises = [];
+  for(let i = 0; i < 1000; i++) {
+    const data = {
+      user_id: faker.random.number({'min':1, 'max':200}), //user_id
+      rated_product: faker.random.number({'min':1, 'max':100}), //rated_product
+      stars_given: faker.random.number({'min':1, 'max':5}) //stars_given
+    }
+    promises.push(ratings_model.createRating(data));
+  }
+  return Promise.all(promises);
```
Comment (mjw0486) [inline database/seed.js:88]: This can be done without promises. Look into async await or bluebird.

File: server/index.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+const config = require(__dirname + '/../config.js');
+const products_controller = require(__dirname + '/controllers/products.js');
+const ratings_controller = require(__dirname + '/controllers/ratings.js');
+
+const app = express();
+const PORT = config.prod_port || config.dev_port;
+
+app.use(bodyParser.json());
+
+app.get('/api/products', products_controller.getAll);
```
Comment (mjw0486) [inline server/index.js:13]: I like the use of external helper functions.

**Review by mjw0486 — COMMENTED** [review]
Looks good, Taylor.

**Pull request merged**